android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt

```diff
@@ -82,9 +85,9 @@ class ModelViewer(val engine: Engine) : android.view.View.OnTouchListener {
     val view: View
     val camera: Camera
     val renderer: Renderer
-    @Entity val light: Int
+    @Entity
```

Comment:
Please revert this change

Reply:
my bad, should I revert all the format changes?

Reply:
No just this one is fine, thanks!